python/ic_azure/azure_client.py

```diff
@@ -32,12 +32,15 @@ def __init__(self, args, api=None, queries=False):
         elif os.getenv("AZURE_CONFIG_PATH") is not None:
             config_file = os.path.join(
                 os.getenv("AZURE_CONFIG_PATH"),
-                args.config
+                args.config,
+                args.config + ".json"
```

Comment:
Including config option twice in file path seems a bit dubious. Couldn't filename be constant in these cases (e.g. azure-config.json)? I don't remember exactly what the spec we decided upon was.

Reply:
Yes it could, but the problem would still remain since we want different folders for each settings...

Reply:
Yes. And args.config specifies the folder name. Maybe I'm not seeing the issue correctly?